cli/args/flags.rs

```diff
@@ -1877,21 +1877,17 @@ Directory arguments are expanded to all contained files matching the glob
     .arg(no_clear_screen_arg())
     .arg(script_arg().last(true))
     .arg(
-      Arg::new("junit")
-        .long("junit")
+      Arg::new("junit-path")
+        .long("junit-path")
```

Comment:
I'm not entirely sure about this name because it sounds like the path to junit, but I don't have a better suggestion.